Total Tickets Available

nicfigu opened the pull request:
quantity to Tickets on Sale, + Total Tickets Available, changed ticket type from any to Ticket, made total_tickets a necessary field in Ticket, changed attendees tab to reflect tickets sold/total tickets available

jventura7 reviewed (commented):
Looks good, a few things: 
- change the placeholder for ticket quantity to “tickets for sale” on create event 
- VendorAssignment.tsx in vendors section of host dashboard still uses quantity, change that to use total tickets

after this you’re good to merge, but if you have time: 
- update edit event to have a field for total tickets

nicfigu commented:
> Looks good, a few things:
> 
> * change the placeholder for ticket quantity to “tickets for sale” on create event
> * VendorAssignment.tsx in vendors section of host dashboard still uses quantity, change that to use total tickets
Done

> after this you’re good to merge, but if you have time:
> 
> * update edit event to have a field for total tickets
Did this on first commit I thought?


added same thing for tables, lmk if it looks good. Also fixed create event to save total_tickets.

jventura7 reviewed (approved):
lgtm
